documentation-site/pages/components/tag.mdx

```diff
@@ -0,0 +1,48 @@
+<!--
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+-->
+
+import Example from '../../components/example';
+import API from '../../components/api';
+import Layout from '../../components/layout';
+
+import Basic from 'examples/tag/basic.js';
+import Kinds from 'examples/tag/kinds.js';
+import CustomColor from 'examples/tag/custom-color.js';
+import Disabled from 'examples/tag/disabled.js';
+
+export default Layout;
+
+# Tag
+
+alskdjflakjsdA design structure to organize and hide content, so as not to overwhelm the user.
+
+### When to use
+
+- laksjdflkj
```

Comment:
🤔 😀

Reply:
:) thank you! I put this here so I don't forget.. guess it only works sometimes